sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.0.2 (2021-06-07)
+- Dependency versions updated.
```

Comment:
You should instead say the below comment, this release is not because of a core update

- Updated `azure-communication-common` version